### DISCUSSION TITLE
Show duration for single specs in HTML-Report

### DISCUSSION THREAD

**Discussion by lumomatone** [badeball/cypress-cucumber-preprocessor#1014] [Q&A]
Hi,

for me in the HTML-Report there is only the total duration from all specs combined visible. Is there a way to also show the duration for each single spec in the HTML-Report?

Kind regards

Lumo

**Comment by VigneshwaranSooriyamoorthy** [comment]
I'm not sure about your way of report generation, but with [cypress-mochawesome-reporter](https://www.npmjs.com/package/cypress-mochawesome-reporter) I can see execution time for each Spec and each Scenario.
